packages/fxa-settings/src/components/GetDataTrio/index.stories.tsx

```diff
@@ -35,7 +35,7 @@ export const SingleCopyButton = () => {
 };
 
 export const SingleCopyButtonInline = () => {
-  const [tooltipVisible, setTooltipVisible] = useState(false);
+  const [, setTooltipVisible] = useState(false);
```

Comment:
Are these the results of a search and replace?

Reply:
They address linter errors. See comment below.